### PR TITLE
Migration Fix - Indexd Ras DRS

### PR DESCRIPTION
**Pull request by krishnaa05**
Link to JIRA ticket if there is one: [GQE-260](https://ctds-planx.atlassian.net/browse/GQE-260)

### Deployment changes

[GQE-260]: https://ctds-planx.atlassian.net/browse/GQE-260?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: gen3-integration-tests/pages/login.py [hunk 1]
```diff
@@ -112,8 +113,10 @@ def ras_login(self, page: Page):
         page.locator(self.RAS_USERNAME_INPUT).fill(os.environ["CI_TEST_RAS_USERID"])
         page.locator(self.RAS_PASSWORD_INPUT).fill(os.environ["CI_TEST_RAS_PASSWORD"])
         ras_login_button.click()
+        time.sleep(3)
```
Comment (haraprasadj) [inline gen3-integration-tests/pages/login.py:116]: can we try and see if this eliminates the need for hard wait - https://playwright.dev/python/docs/api/class-locatorassertions#locator-assertions-to-be-enabled

Comment (krishnaa05) [reply gen3-integration-tests/pages/login.py:116]: I dont think its the problem of button being enabled. The playwright was proceeding without even going into the if statement. I did try to increase the timeout but that didnt help. Tried the below code as well:
if locator.is_enabled():

Finally when time.sleep was used, it went inside the if statement and clicked on the button

Comment (haraprasadj) [reply gen3-integration-tests/pages/login.py:116]: hmm, I can't see how a timeout solves that

Comment (krishnaa05) [reply gen3-integration-tests/pages/login.py:116]: I understand, even I am not quite sure why the timeout didnt work and time.sleep did work but this solution works. I can try exploring more on it to see

**Review by haraprasadj — APPROVED** [review]

**Pull request merged**